hermes-console/static/partials/topic.html

```diff
@@ -195,28 +195,9 @@ <h4>
                     Offline clients
                 </h4>
             </div>
-            <table class="table table-bordered">
-                <thead>
-                    <tr>
-                        <th></th>
-                        <th>User</th>
-                        <th>Last access date</th>
-                        <th>Owners</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr ng-repeat="client in clients">
-                        <td>{{$index + 1}}</td>
-                        <td>{{client.user}}</td>
-                        <td>{{client.lastAccess}}</td>
-                        <td>{{client.owners.join(", ")}}</td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
-
-        <div ng-show="offlineClientsFetching" class="text-center">
-            <h1><i class="fa fa-circle-o-notch fa-spin"></i></h1>
+            <div class="list-group-item">
+                <iframe height="400" ng-src="{{trustSrc(iframeSource)}}" style="border: none;" width="100%"></iframe>
```

Comment:
What will happen when fetching iframe source will throw exception (for example `OfflineClientsServiceAbsentException`)? From code it looks like nothing here will be rendered. Can you add alert here in failing scenario?

Reply:
When there is absent `OfflineClientsService` user should keep `config.offlineClientsEnabled` property set to false, if so the entire part of ui with offline clients will not be rendered. In my opinion thats not necessary to add alert in case of failing request, also we don't do that either while checking blacklistedStatus or fetching subscriptions for topic.